spec/appendix.md

```diff
@@ -1,12 +1,58 @@
 ## Appendix
 
+### Trust Over IP Glossary - Controlled Terms 
+
+NOTE: 
+* The following set of identifiers are from the current [ToIP Glossary](https://docs.google.com/document/d/1fZByfuSOwszDRkE7ARQLeElSYmVznoOyJK4sxRvJpyM/edit?usp=sharing) (Google Doc). These definitions need to be moved over to the (under development) ToIP Terms tool (aka TEv2).
+* TODO: not all ToIP definitions are fully linked here. This whole section is more of an exemplar as of the time of writing.
```

Comment:
Presumably you mean “not all the relevant ToIP definitions…”.  Presumably, the ToIP Glossary is much bigger than this.

Reply:
@darrellodonnell I believe this TODO line should be removed. The addition of any other terms can be included in a future PR.

Reply:
noted - I have made edits to remove the NOTE as well as removed a few definitions that go too far (i.e. really belong in an external glossary). Added issue to GH as well: https://github.com/trustoverip/tswg-did-method-webs-specification/issues/121 

update coming shortly